examples/training/classification/imagenet/basic_training.py

```diff
@@ -21,6 +21,7 @@
 
 import sys
 
+import numpy as np
```

Comment:
NIT:
without side effects we could remove numpy with
```python
import math as m
pi = tf.constant(m.pi)
```

Reply:
Sure! I can update it :)
Are there any benefits of `m.pi` vs `np.pi`? Lower overhead perhaps?

Reply:
I don't think we have a specific overhead.. it was just nit to not import the library just for a single call.